shared/stellar/index.stories.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react'
+import {Text} from '../common-adapters'
+import {storiesOf} from '../stories/storybook'
+
+const load = () => {
```

Comment:
Let's add a comment that these should actually be implemented in their own files   Aka walletlist/index. Stories. Js